spec/controllers/admin/countries_controller_non_admin_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+#This file tests that non logged in users and non admin schools can not perform admin compromised actions
+RSpec.describe Admin::CountriesController, type: :controller do
+  let(:country) { create(:country) }
+
+  describe "non logged in school" do
+    it "#index redirects to sign_in" do
+      get :index
+      expect(response).to redirect_to new_school_session_url
+    end
+
+    it "#create redirects to sign_in" do
+      expect {
+        post :create, country: attributes_for(:country)
+      }.to_not change{Country.count}
+      expect(response).to redirect_to new_school_session_url
+    end
+
+    it "#new redirects to sign_in" do
+      get :new
+      expect(response).to redirect_to new_school_session_url
+    end
+
+    it "#edit redirects to sign_in" do
+      get :edit, id: 1
+      expect(response).to redirect_to new_school_session_url
+    end
+
+    it "#show redirects to sign_in" do
```

Comment:
the scenario should have a better description. You can have the `#show` in a describe block, the it should state the scenario. In this case something like "redirects to the sign in page if the school isn't registed" would be better. You can also wrap "unregistered school" in its own describe block.